### PR TITLE
[skip-ci] Small updates to RDF docs

### PR DESCRIPTION
**Pull request by vepadulano**

### CODE REVIEW

File: tree/dataframe/src/RDataFrame.cxx [hunk 1]
```diff
@@ -620,7 +645,7 @@ RDataFrame operations currently work with this package. The subset that is curre
 - Fill
 - Filter
 - Graph
-- Histo[1,2,3]D
+- Histo[1,2,3,N]D
 - HistoND
```
Comment (ikabadzhov) [inline tree/dataframe/src/RDataFrame.cxx:649]: then next line should go

**Review by ikabadzhov — APPROVED** [review]
lgtm, but see comment `HistoND` is now mentioned twice!

File: tree/dataframe/src/RDataFrame.cxx (hunk 1 above)
Comment (vepadulano) [reply tree/dataframe/src/RDataFrame.cxx]: I actually didn't read it, I think it is fine as it it, in the latest commit I removed this addition

Comment (vepadulano) [reply tree/dataframe/src/RDataFrame.cxx]: Thanks for catching it!

File: tree/dataframe/src/RDataFrame.cxx [hunk 2]
```diff
@@ -629,6 +629,8 @@ RDataFrame operations currently work with this package. The subset that is curre
 - Redefine
 - Snapshot
 - Sum
+- Systematic variations: Vary and VariationsFor
+- Parallel submission of distributed graphs: RunGraphs
```
Comment (eguiraud) [inline tree/dataframe/src/RDataFrame.cxx:658]: RunGraphs and VariationsFor are a bit special as they are not RDF methods but rather free functions. Can we link to their docs from here?

**Review by eguiraud — APPROVED** [review]
Thank you very much!

File: tree/dataframe/src/RDataFrame.cxx (hunk 2 above)
Comment (vepadulano) [reply tree/dataframe/src/RDataFrame.cxx]: Done!

**Pull request merged**